kv/bucket.go

```diff
@@ -173,6 +141,30 @@ func (s *Service) FindBucketByName(ctx context.Context, orgID influxdb.ID, n str
 	return b, err
 }
 
+func (s *Service) findSystemBucket(n string) (*influxdb.Bucket, error) {
+	switch n {
+	case "_tasks":
+		return &influxdb.Bucket{
+			ID:              influxdb.TasksSystemBucketID,
+			Name:            "_tasks",
+			RetentionPeriod: time.Hour * 24 * 3,
+			Description:     "System bucket for task logs",
+		}, nil
+	case "_monitoring":
+		return &influxdb.Bucket{
+			ID:              influxdb.MonitoringSystemBucketID,
+			Name:            "_monitoring",
+			RetentionPeriod: time.Hour * 24 * 7,
+			Description:     "System bucket for monitoring logs",
+		}, nil
+	default:
+		return nil, &influxdb.Error{
+			Code: influxdb.ENotFound,
+			Msg:  fmt.Sprintf("bucket %q not found", n),
```

Comment:
same, change it to system bucket %q not found, or add an Op "kv/findSystemBucket"